beacon-chain/blockchain/service.go

```diff
@@ -226,6 +196,22 @@ func (c *ChainService) updateHead(slot uint64) {
 	c.candidateCrystallizedState = nilCrystallizedState
 }
 
+// doesPoWBlockExist checks if the referenced PoW block exists.
+// If the node is not connected to a PoW chain, defaults to true.
```

Comment:
I think this is a dangerous pattern. Can we change it to default to false and log an error?

Reply:
I'm not against it. @nisdas voiced similar concerns as well. The line was originally added [here](https://github.com/prysmaticlabs/prysm/commit/e1f727cbb2bf3c2d569df1acc159ab4adf6fca03#diff-c5dd5428da96b5d8884a099d3a3ecf44R190). @terenc3t , any objections?

btw, `isValidator = c.web3Service != nil`, so these two lines are the same thing.